doc/secciones/03_estado_del_arte.tex

```diff
@@ -1,5 +1,31 @@
 \chapter{Estado del arte}
-En los últimos años ha habido una revolución en el análisis de la organización 
+
+La historia del análisis de fútbol comienza en 1950, cuando Charles Reep\cite{reep-bio} empezó a 
+registrar sistemáticamente los eventos que tenían lugar a lo largo de un partido de Swindon Town. Con ello, no pretendía 
+únicamente tener detalle de lo que había ocurrido, sino también saber por qué pasaba y qué podían aprender los 
+equipos de los datos, para mejorar su juego. Tras décadas, llegó a la conclusión de que el mayor número 
+de goles resultaba de posesiones de tres pases o menos, por lo que aludía que los equipos debían simplificar 
+sus tácticas para llegar a portería más rápido.
+
+El problema de Reep no fue su recolección de datos, sino su análisis; podría haberse preguntado sobre 
+la tasa de goles en posesiones de varias distancias, o cómo se desarrollaron exactamente esas 
+posesiones de tres pases. Su trabajo mostró que "El 60 por ciento de todos los movimientos de gol 
+comienzan a 32 metros de la portería de un oponente".
+
+En 1996, \href{https://es.wikipedia.org/wiki/Opta_Sports}{Opta} empezó a acumular datos de partidos para la 
+\textit{Premier League} inglesa. Ello incluía todas las estadísticas a las que un aficionado al fútbol está 
+acostumbrado: número de pases, número de regateos, distancia recorrida, por ejemplo. Este podría considerarse 
+el punto de comienzo del análisis de fútbol moderno. 
+
+En 2003, Michael Lewis lanzó el libro \href{https://en.wikipedia.org/wiki/Moneyball}{Moneyball}, el cual tuvo una gran 
+influencia también fuera de Estados Unidos. Trata sobre cómo la oficina de \textit{Oakland Athletics} convertió 
+un equipo de béisbol de bajo presupuesto en uno de los mejores, usando estadísticas para reclutar a jugadores 
+con habilidades hasta entonces minusvaloradas. El lanzamiento del libro fomentó que se generaran preguntas en 
+torno al uso de datos en el fútbol. Un par de irracionalidades del mercado de fichajes
+quedaron expuestas, sobre todo la relación entre salario y duración de la carrera de los porteros
+y delanteros y un extraño sesgo hacia la contratación de jugadores más altos.
+
+Desde 2003 hasta ahora ha habido una revolución en el análisis de la organización 
```

Comment:
2003, *el año de publicación de este libro*,...

También con esto indicas que hay una relación causal, y no la veo. Durante esa época hubo muchos intentos de relacionar redes complejas y fútbol, mira este artículo (un póster) de 2005

@inproceedings{lee2005passes,
  title={Who passes to whom: Analysis of optimal network structure in soccer matches},
  author={Lee, Jegoo and Borgatti, Stephen P and Molina, Jose Luis and Guervos, Juan J Merelo},
  booktitle={XXV International Sunbelt Social Network Conference},
  year={2005}
}
 Ya en el 2004 se lanzó un desafío en la lista redes para predecir el resultado de la Eurocopa usando redes sociales. Yo creo que se debe más al estallido del análisis de redes sociales que al libro, aunque todo puede ayudar.